x-pack/plugins/actions/server/builtin_action_types/webhook.ts

```diff
@@ -85,6 +85,15 @@ function validateActionTypeConfig(
   configurationUtilities: ActionsConfigurationUtilities,
   configObject: ActionTypeConfigType
 ) {
+  let url: URL;
+  try {
+    url = new URL(configObject.url);
+  } catch (err) {
+    return i18n.translate('xpack.actions.builtin.webhook.webhookConfigurationErrorNoHostname', {
+      defaultMessage: 'error configuring webhook action: unable to parse host name from Url',
```

Comment:
The message is too specific, and may not be relevant if there is something wrong with the URL not involving the host name.

It's also possible that `configurationUtilities.ensureWhitelistedUri()` - called below this - may end up doing doing this check - I hope it would in fact - but maybe there's some reason it doesn't.